libraries/apollo-mockserver/src/commonMain/kotlin/com/apollographql/apollo3/mockserver/MockServer.kt

```diff
@@ -251,38 +251,59 @@ fun MockServer(): MockServer = MockServerImpl(
     QueueMockServerHandler(),
     true,
     TcpServer(0),
-    null)
+    null
+)
 
 @Deprecated("Use MockServer.Builder() instead", level = DeprecationLevel.ERROR)
 @ApolloDeprecatedSince(ApolloDeprecatedSince.Version.v4_0_0)
 fun MockServer(handler: MockServerHandler): MockServer =
-    MockServerImpl(
-        handler,
-        true,
-        TcpServer(0),
-        null)
+  MockServerImpl(
+      handler,
+      true,
+      TcpServer(0),
+      null
+  )
 
 @Deprecated("Use enqueueString instead", ReplaceWith("enqueueString(string = string, delayMs = delayMs, statusCode = statusCode)"), DeprecationLevel.ERROR)
 @ApolloDeprecatedSince(ApolloDeprecatedSince.Version.v4_0_0)
 fun MockServer.enqueue(string: String = "", delayMs: Long = 0, statusCode: Int = 200) = enqueueString(string, delayMs, statusCode)
 
-fun MockServer.enqueueString(string: String = "", delayMs: Long = 0, statusCode: Int = 200, contentType: String = "text/plain") {
+fun MockServer.enqueueString(string: String, delayMs: Long = 0, statusCode: Int = 200, contentType: String = "text/plain") {
   enqueue(MockResponse.Builder()
       .statusCode(statusCode)
       .body(string)
       .addHeader("Content-Type", contentType)
       .delayMillis(delayMs)
-      .build())
+      .build()
+  )
 }
 
-fun MockServer.enqueueGraphQLString(string: String) {
+fun MockServer.enqueueError(statusCode: Int = 200) {
```

Comment:
Maybe the default should be 500?

```suggestion
fun MockServer.enqueueError(statusCode: Int = 500) {
```

Reply:
Urg 🤦  Good catch. It should be a manfatory parameter.